### ISSUE TITLE
Guidance for onboarding new co-organizers

### ISSUE THREAD

**Issue by maelle** [rladies/rladiesguide#18]
@yabellini @hfrick I am trying to add this, can you tell me whether this is correct?

Once one has found a co-organizer (yay!) the steps are

* Update the meetup list of co-organizers
* Make a PR to https://github.com/rladies/starter-kit/blob/master/Current-Chapters.csv (is this optional?)
* Email email@rladies.org to ask with the address of the new organizer, so that they start receiving emails too.
* Email info@rladies.org to ask for the new organizer to be added to the organizers slack. (Hannah, would another email address make sense?)

Point the co-organizer to guide.rladies.org

**Comment by yabellini** [comment]
The point you listed are ok.
The formal process now it is that you inform to info@rladies.org about the new organizer so the leadership team can add to Slack, the organizer update the Current Chapter with their name (I help with that task in case git isa problem) and ask by email to add to the forward's list of the email chapter.

**Comment by maelle** [comment]
should one still first update the list on meetup?
and do you agree that one sends the email both to info@rladies.org and email@rladies.org?

Thank you!

**Comment by yabellini** [comment]
I think the first step up to inform Global Team that there is a new member. But you can do all this thing together.  
As I see the process, it would be better if they only have to send an email to info@rladies.org and the global team triggers a process where internally we notify ourselves what we have to do (add to Slack, add to GitHub, add to the forward list of the email) without the need for the organizer to be sending several different emails.  that process also explain how to add to meetup and update Current Chapter (Global Team can help with that also).

**Comment by maelle** [comment]
Do I follow correctly that the process you are describing does not exist yet? 

So in the end the guidance would be "add the organizer to Meetup yourself and inform the global team by emailing `new-chapter@rladies.org`" or whatever email address @hfrick & @PatriLoto end up requesting (by email :wink:).

But we'd also need a process, documented in the guide, for the onboarding team to deal with such requests. 
* Slack
* GitHub
* email
* current-chapters.csv

Is this something that needs to happen in the context of other changes or can it happen now?
In the meantime, what should the guidance for organizers be?

**Comment by maelle** [comment]
On meetup @patriloto can you please confirm that any organizer of a chapter can add new organizers themselves without having to ask you or other volunteers in charge of the Meetup pro account?

**Comment by yabellini** [comment]
Yes, you are correct, that process doesn't exist yet. Leadership Team is the only one who can read info@rladies.org so when a person asks me to add someone to slack, Github, and/or to the email,  I ask if they have already notified info@rladies.org if they say no, I ask them to send the email. If they tell me yes, I take their word for it and add to Slack/email/Github.

**Comment by rivaquiroga** [comment]
Hi, everyone! 
The "How to onboard organizers?" section of R-Ladies organizational guidelines points to this issue. 
If I understand correctly the previous conversation here, the stepts are:

1. write to info@rladies.org to inform about the new organizer
2. add them to the current chapter csv (but... Is this something I can do in the meantime, or I have to wait for some kind of answer?)
3. add them to Meetup as a co-organizer
4. write to email@rladies.org to ask for the email account

Is that right? 
Thanks!

**Comment by yabellini** [comment]
The Global Team will answer you asking the new organizer fill a form so we can invite to the slack, give github permission, etc.
We no longer provide personal emails so no need to do point 4.
You don't need to wait for the answer to add them to the csv.

**Comment by rivaquiroga** [comment]
thanks!

**Comment by maelle** [comment]
Is the current process stable enough to be documented or should I wait a bit?
